docs/intro-to-coalton.md

```diff
@@ -4,9 +4,55 @@ Coalton is a statically typed language that is embedded in, and compiles to, Com
 
 This document is aimed toward people who are already familiar with functional programming languages.
 
+## Systems and Packages
+
+Coalton uses ordinary Common Lisp packages (and ASDF systems) to organize code. If you're starting a new project, add `#:coalton` to your ASD's `:depends-on` list.
```

Comment:
I've been on record on numerous occasions advocating to use `:keywords` for package names in e.g. `defpackage` forms, and `#:uninterned-symbols` for non-package names like those in `:export` or `:import-from`. I understand this is somewhat controversial (and I also don't care enough to make a stink), but I'd like to put on record (here, and everywhere else I can) that the only meaningful difference between keywords and uninterned symbols in a `defpackage`-ey context is how SL[Y|IME] autocomplete treats them, and I believe that it's generally desirable for autocomplete to suggest package names.

Reply:
It's interesting that I basically have the opposite opinion: I only like keyword completion for keyword arguments, and at no other time. The fact that every (obviously interned) keyword autocompletes everywhere vexes me.

If it weren't for Allegro/modern-mode/etc., I'd normally advocate for `"ALL-CAPS"` strings for everything.